Update faqs 20250922 Geographic metadata FAQ & fix typos

DeondeJager opened the pull request.

DeondeJager committed:
Update faq.md Geographic metadata & typos

- Filled-in geographic metadata FAQ.
- Fixed a couple of typos.

DeondeJager committed:
Update faq.md Fix ticks around term names.

jfy133 reviewed (commented):
Looks good as first pass! Mostly maybe consider removing a bit of the detail that is already in the term itself, and give examples with the aDNA specific implementation

DeondeJager committed:
Update faq.md

Updated geography FAQ by removing some details and adding an example.

jfy133 commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,20 @@ _TODO_
 
-_TODO_
+-   **Geographic location (latitude and longitude)**:
+    -  The MIxS term [`Geographic location (latitude and longitude)`](https://genomicsstandardsconsortium.github.io/mixs/0000009/) is used to record the coordinates in decimal degrees, limited to 8 decimal points and separated by a single whitespace (e.g. `-39.27774231 175.61028236`). While limited to 8 decimal points, you will rarely need such precision, unless you've extracted DNA from a [speci ...
```
The grain of sand thing should maybe go as a footnote using `[^1]` and `[^1]: description` syntax

jfy133 reviewed (commented):
Additional minor comments

DeondeJager committed:
Update Geographic location faq.md

Need to rewrite this FAQ once site location terms have been added (see https://github.com/MIxS-MInAS/extension-ancient/issues/45#issue-3803821736)

DeondeJager committed:
Move grain of sand to footnote.

As suggested by James [here](https://github.com/MIxS-MInAS/MIxS-MInAS.github.io/pull/14/changes#r2681621333)

jfy133 commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
 ## How to fill in geographic metadata
+There are two primary ways of providing the geographic location of a sample with existing MIxS terms: `Geographic location (latitude and longitude)` and `Geographic location (country and/or sea,region)`. 
+These fields are used to record the museum location if the specimen originated from a museum, because this is where the sampling with intention to sequence occurred (see the ENA [FAQs](https://ena-docs.readthedocs.io/en/latest/faq/spatiotemporal-metadata.html#i-have-an-ancient-dna-sample-how-do-i-provide-the-spatiotemporal-information-for-my-sample-which-location-and-date-should-be-reported)). 
+If the specimen was excavated from an archaeological/palaeontlogical site with the intention to sequence (without first going via a museum) then the site information goes in the above terms.
```
I would flip this:

1. By default we assume samples were sampled directly from the environment/burial context
2. How IF the samples have been removed from their natural environment/burial context and are stored and/or were sampled from a collection or archive, the ENA guidance is that these geographic information terms should represent the collection or archive. 

i.e., the collections is the _exception_ not the rule

DeondeJager replied:
```suggestion
These fields are used to record the location that a specimen/sample was collected from with the intention of sequencing.

If the specimen was excavated from an archaeological/palaeontlogical site with the intention to sequence (without first going via a museum) then the site information goes in the above terms.

However, if the specimen was excavated/retrieved and then deposited in a museum (or other) collection and only after that was a sample taken explicitly for sequencing, then the locality details of the museum/collection must be recorded in these fields! See the ENA [FAQs](https://ena-docs.readthedocs.io/en/latest/faq/spatiotemporal-metadata.html#i-have-an-ancient-dna-sample-how-do-i-provide-the-spatiotemporal-information-for-my-sample-which-location-and-date-should-be-reported) regarding this. 

```

jfy133 commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
+There are two primary ways of providing the geographic location of a sample with existing MIxS terms: `Geographic location (latitude and longitude)` and `Geographic location (country and/or sea,region)`. 
+These fields are used to record the museum location if the specimen originated from a museum, because this is where the sampling with intention to sequence occurred (see the ENA [FAQs](https://ena-docs.readthedocs.io/en/latest/faq/spatiotemporal-metadata.html#i-have-an-ancient-dna-sample-how-do-i-provide-the-spatiotemporal-information-for-my-sample-which-location-and-date-should-be-reported)). 
+If the specimen was excavated from an archaeological/palaeontlogical site with the intention to sequence (without first going via a museum) then the site information goes in the above terms.
+Both of these are required fields and we suggest filling in at both terms with as precise information as possible. 
```
```suggestion
Both of these are required fields and we suggest filling in at both the coordinate and location name terms with as precise information as possible. 
```

jfy133 reviewed (commented):
I'm still finding  this section a bit confusing to follow...

The way I would structure it put it is something more like:

1. MIxS and MInAS has two ways of representing geographical information about a sample
2. These two ways cover where the sample normally is based at the time of sampling (e.g. museum), and where the original geographical 'source' the sample was from (the original burial site, or natural location)
3. For the former, use the MIxS terms XYZ... (with specific guidance for each term, e.g. lat_lon precision)
4. For the latter, use the MInAS terms `orig_...` (with specific guidance for each term, e.g. lat_lon precision)
5. If you directly sampled the ancient material at the burial location, the information will be the same

General thing: try to avoid bullet point lists when you have long text

And I guess now we have the new terms you can add more examples too :)

DeondeJager commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
-_TODO_
+If the specimen is from a museum, we still need to record the original location of the specimen, which is the site from which it was excavated. 
+In this case, use the MInAS site terms: 
+    -    _insert site_location term_
```
```suggestion
    -    [original site location](https://genomicsstandardsconsortium.github.io/mixs/0001357/)
```

DeondeJager commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
+If the specimen is from a museum, we still need to record the original location of the specimen, which is the site from which it was excavated. 
+In this case, use the MInAS site terms: 
+    -    _insert site_location term_
+    -    _insert site_lat_lon tarm_
```
```suggestion
    - [original geographic location (latitude)](https://genomicsstandardsconsortium.github.io/mixs/0001358/)
    - [original geographic location (longitude)](https://genomicsstandardsconsortium.github.io/mixs/0001359/)
    
```

DeondeJager commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
+-   **Geographic location (country and/or sea,region)**:
+    -   The MIxS term [`Geographic location (country and/or sea,region)`](https://genomicsstandardsconsortium.github.io/mixs/0000010/) is used to record this information in the format `Country:Region, Locality`.
+    -   Country or sea names should be chosen from the [INSDC country list](http://insdc.org/country.html), or the [GAZ ontology](http://purl.bioontology.org/ontology/GAZ)
+    -   This term is not intended for the archaeological/palaeontological/ecological site name (if relevant). Instead, include the site name under the MInAS-specific term `Name of site or location where sample was originated`.
```
```suggestion
    -   This term is not intended for the archaeological/palaeontological/ecological site name (if relevant). Instead, include the site name under the MInAS-specific term `Name of site or location where sample was originated`, see [here](https://genomicsstandardsconsortium.github.io/mixs/0001356/).
```

DeondeJager replied:
```suggestion
    -   The term `Geographic location (country and/or sea,region)` is not intended for the archaeological/palaeontological/ecological site name (if relevant). Instead, include the site name under the MInAS-specific term `Name of site or location where sample was originated`.
```

DeondeJager commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
+    -    _insert site_lat_lon tarm_
+    -    `Name of site or location where sample was originated`, which should be used to record the name of the archaeological/palaeontological site.
+
+-   **Geographic location (latitude and longitude)**:
```
```suggestion
-   **Some general tips**:
```

DeondeJager commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
+    -    `Name of site or location where sample was originated`, which should be used to record the name of the archaeological/palaeontological site.
+
+-   **Geographic location (latitude and longitude)**:
+    -  The MIxS term [`Geographic location (latitude and longitude)`](https://genomicsstandardsconsortium.github.io/mixs/0000009/) is used to record the coordinates in decimal degrees, limited to 8 decimal points and separated by a single whitespace (e.g. `-39.27774231 175.61028236`). While limited to 8 decimal points, you will rarely need such precision[^1]. See [here](https://en.wikipedia.org/w ...
```
```suggestion
    -  Coordinates reported in decimal degrees are limited to 8 decimal points and _separated by a single whitespace_ (e.g. `-39.27774231 175.61028236`). 
    - While limited to 8 decimal points, you will rarely need such precision[^1]. See [here](https://en.wikipedia.org/wiki/Decimal_degrees) to guide your decision on the appropriate level of precision to report.
```

DeondeJager commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -23,12 +23,31 @@ _TODO_
+
+-   **Geographic location (country and/or sea,region)**:
+    -   The MIxS term [`Geographic location (country and/or sea,region)`](https://genomicsstandardsconsortium.github.io/mixs/0000010/) is used to record this information in the format `Country:Region, Locality`.
+    -   Country or sea names should be chosen from the [INSDC country list](http://insdc.org/country.html), or the [GAZ ontology](http://purl.bioontology.org/ontology/GAZ)
```
```suggestion

```

DeondeJager committed:
Apply suggestions from code review

Co-authored-by: James A. Fellows Yates <jfy133@gmail.com>
Co-authored-by: Deon de Jager <dejager4@gmail.com>

DeondeJager commented on docs/documentation/user-documentation/faq.md:
```diff
@@ -24,11 +24,38 @@ _TODO_
+    -   `orig_site_loc`, which should be the local geographic name (e.g. closest town/region/named area on a map)
+    -   `orig_site_lat`, which should be the latitude of the original burial site/natural environment 
+    -  `orig_site_lon`, which should be the longitude of the original burial site/natural environment
+    -   `orig_site_name`, which should be used to record the name of the archaeological/palaeontological site.
```
```suggestion
    - [original geographic location (latitude)](https://genomicsstandardsconsortium.github.io/mixs/0001358/), which should be the latitude of the original burial site/natural environment
    - [original geographic location (longitude)](https://genomicsstandardsconsortium.github.io/mixs/0001359/), which should be the longitude of the original burial site/natural environment
    - [original site location](https://genomicsstandardsconsortium.github.io/mixs/0001357/), which should be the local geographic name (e.g. closest town/region/named area on a map)
    - [name of site or location where sample originated](https://genomicsstandardsconsortium.github.io/mixs/0001356/), which should be used to record the name of the archaeological/palaeontological site.
```

DeondeJager committed:
Apply suggestions from code review

Co-authored-by: James A. Fellows Yates <jfy133@gmail.com>
Co-authored-by: Deon de Jager <dejager4@gmail.com>

DeondeJager committed:
Merge branch 'master' into Update-FAQs-20250922
